Sources/TuistCore/Models/Template.swift

```diff
@@ -0,0 +1,53 @@
+import Basic
+
+public struct Template: Equatable {
+    public static func == (lhs: Template, rhs: Template) -> Bool {
```

Comment:
Considering `Template` is a struct and that all the attributes seem equatable, I believe you can get rid of this implementation.

Reply:
Compiler was not my good friend yesterday, but I'll try my chances again (I don't know why it kept complaining about this missing function)

Reply:
Make sure that all the attribute types conform to equatable. If they do, the compiler should be happy.